Issue 2442 Button shortcuts

Marko64 opened the pull request:
Fixes #2442
- Edited defaults for Preset 3 and Preset 5

Marko64 committed:
Fixed button shortcutes

- Preset 3 and Preset 5

Naaaaiix reviewed (commented):
Still buggy for me:

https://user-images.githubusercontent.com/50477222/148790630-3c347d10-d882-46d9-9ad8-dbad8e7b2bcb.mp4

Marko64 committed:
Merge branch 'master' of https://github.com/yeahcan/maxi-blocks into issue-2442-button-shortcuts

Marko64 committed:
Update defaults.js

Marko64 committed:
Merge branch 'master' of https://github.com/yeahcan/maxi-blocks into issue-2442-button-shortcuts

Marko64 committed:
Defaults.js update

- Removed background, border and box-shadow properties where they were not needed.
